code/backend/pages/03_Delete_Data.py

```diff
@@ -78,7 +80,23 @@ def delete_files(files):
     st.success("Deleted files: " + str(files_to_delete))
 
 
+def get_filename(file: str) -> str:
+    """
+    Parses a file path in the format:
+    /container_name/file_name
+    to return the file name.
+
+    Args:
+        file (str): The file path of the file to delete.
+
+    Returns:
+        filename (str): The filename part of the file path
+    """
+    return file.split("/")[2]
```

Comment:
What is the behavior if, for some reason, the split or the [2] doesn't work? Will the code throw an error and never get to the search_client.delete_documents()?